ocaml/xapi/xapi_vif.ml

```diff
@@ -116,3 +162,23 @@ let add_ipv6_allowed ~__context ~self ~value =
 let remove_ipv6_allowed ~__context ~self ~value =
 	change_locking_config ~__context ~self ~licence_check:false
 		(fun () -> Db.VIF.remove_ipv6_allowed ~__context ~self ~value)
+
+let add_to_static_ip_setting ~__context ~self ~key ~value =
+	change_locking_config ~__context ~self ~licence_check:false
```

Comment:
This should _not_ be `change_locking_config`, which for the VIF port locking feature.